tests/LiquiditySwapV2.t.sol

```diff
@@ -276,64 +388,128 @@ contract LiquiditySwapAdapterV2 is BaseTest {
 
     IERC20Detailed(collateralAssetAToken).approve(
       address(liquiditySwapAdapter),
-      collateralAmountToSwap
+      oldCollateralAssetATokenBalanceBefore
```

Comment:
better to use `collateralAmountToSwap` no? which is the value used for the psp call